### DISCUSSION TITLE
Please recommend me larger than 4.0 inch IPS displays!

### DISCUSSION THREAD

**Discussion by dexter323i** [Bodmer/TFT_eSPI#3543] [Compatible displays and setup files]
Hi!

Can you please recommend me larger IPS displays compatible with this library? I've been using 3.5 inch with 8 bit parallel interface, and ESP32, but I'd like to go bigger.
I've found 4.0 inch with ILI9488 320*480, but nothing larger yet. Something around 4.3 inch or 5 inch would be really nice!

Dexter

**Comment by IanBUK** [comment]
> Hi!
> 
> 
> 
> Can you please recommend me larger IPS displays compatible with this library? I've been using 3.5 inch with 8 bit parallel interface, and ESP32, but I'd like to go bigger.
> 
> I've found 4.0 inch with ILI9488 320*480, but nothing larger yet. Something around 4.3 inch or 5 inch would be really nice!
> 
> 
> 
> Dexter

Head on over to BuyDisplay.com. I bought myself a 7" 800x480 display from them, with shield, a couple of years ago for around 70USD. The same model is cheaper now. Mine has an ssd1963 driver. Looking at the site, they've changed the touchscreen controller. 

https://www.buydisplay.com/7-inch-arduino-touch-screen-shield-ssd1963-library-for-mega-due
